src/lib/types/file-type.ts

```diff
@@ -0,0 +1,83 @@
+// Reference: https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types
+export enum FileMimeTypes {
```

Comment:
This is being added here but is probably worth thinking if it should be either smaller and for the specificity of the MultiDropzone component or even created and imported in another repo like https://github.com/getPopsure/public-models

Reply:
I'd argue that we could start with a smaller list and add more types as we need them.
Let's see what the other teammates have to say about this.

Reply:
I think we can limit it to the specific file types that we / ops team want then just add the ones we want later. Since the defined type is specific to this upload component, I think it makes that it lives here and then we can just export it. wdyt?

Reply:
That makes sense to me. I just updated the code to use a smaller file that defined inside of MultiDropzone types.

Some notes on this update:
- I made `type` on UploadedFile not required as it is not being used anymore but I don't want to introduce any breaking changes,
- The only case where we might need additional FileTypes is if we add a custom `accept` as the component accepts any filetypes by default.